scripts/required_fields.yaml

```diff
@@ -0,0 +1,261 @@
+article:
+  required:
+    - author
+    - title
+    - journal
+    - year
+  wanted:
+    - volume
+    - number
+    - pages
+    - month
+    - url
+    - doi
+  optional:
+    - note
+book:
+  required:
+    - - author
+      - editor
```

Comment:
document that this syntax has alternative semantics

Reply:
done